src/policy/rbf.h

```diff
@@ -35,4 +35,19 @@ enum class RBFTransactionState {
 RBFTransactionState IsRBFOptIn(const CTransaction& tx, const CTxMemPool& pool) EXCLUSIVE_LOCKS_REQUIRED(pool.cs);
 RBFTransactionState IsRBFOptInEmptyMempool(const CTransaction& tx);
 
+/** Get all descendants of setIterConflicting. Also enforce BIP125 Rule #5, "The number of original
+ * transactions to be replaced and their descendant transactions which will be evicted from the
+ * mempool must not exceed a total of 100 transactions." Quit as early as possible. There cannot be
+ * more than MAX_BIP125_REPLACEMENT_CANDIDATES potential entries.
+ * @param[in]   setIterConflicting  The set of iterators to mempool entries.
+ * @param[out]  err_string          Used to return errors, if any.
+ * @param[out]  allConflicting      Populated with all the mempool entries that would be replaced,
+ *                                  which includes descendants of setIterConflicting. Not cleared at
+ *                                  the start; any existing mempool entries will remain in the set.
```

Comment:
What do you mean by "Not cleared at the start; any existing mempool entries will remain in the set" ?

If the replacement is successful,  `allConflicting` are going to be removed in `Finalize`. For package support, I think this operation we'll remove the per-package tx `allConflicting` set all at once. If `PreChecks` or any new other package policy checks as failed, don't modify mempool state.

Reply:
It's similar to other mempool functions where you can pass in a `setEntries` and it will `insert` entries, but not expect it to be empty beforehand, so you can call it multiple times and end up with an aggregated set. I'm expecting to use this in package RBF. It has nothing to do with what's going to end up in the mempool; this is just a helper function.

The reason I document it here is I think it's a relevant detail in the interface. It's the caller's responsibility to clear the set beforehand.